### PR TITLE
ci: use hashes in pip install statements in release.sh

### PR DESCRIPTION
**Pull request by parthea**
chore: add owlbot.py to copy templated release.sh

### CODE REVIEW

**Comment by parthea** [comment]
The Kokoro CI failure is unrelated to the changes in this PR

File: .kokoro/release.sh [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
-
-# Copyright 2018 Google LLC
+# Copyright 2020 Google LLC
```
Comment (dandhlee) [inline .kokoro/release.sh:2]: No need to update this.

Comment (dandhlee) [reply .kokoro/release.sh:2]: Nevermind this is a templated update, fine to leave it as is.

**Review by dandhlee — APPROVED** [review]
LGTM.

**Comment by parthea** [comment]
Admin merging as the Kokoro CI failures are unrelated.

**Pull request merged**